tests/test_model.py

```diff
@@ -166,16 +166,8 @@ def test_segment_oatcls_scte35_dumps():
     # Only insert OATCLS-SCTE35 at cue out
     cue_out_line = (
         '#EXT-OATCLS-SCTE35:/DAlAAAAAAAAAP/wFAUAAAABf+//wpiQkv4ARKogAAEBAQAAQ6sodg==\n'
-        '#EXT-X-CUE-OUT'
     )
-    assert cue_out_line in result
-    
-    # Don't insert it for continued cue outs
-    cue_out_cont_line =  (
-        '#EXT-OATCLS-SCTE35:/DAlAAAAAAAAAP/wFAUAAAABf+//wpiQkv4ARKogAAEBAQAAQ6sodg==\n'
-        '#EXT-X-CUE-OUT-CONT'
-    )
-    assert cue_out_cont_line not in result
+    assert result.count(cue_out_line) == 1
```

Comment:
The earlier version of this test meant to test that the `EXT-OATCLS-SCTE35` tag wasn't repeated and didn't need to dictate the exact ordering of tags.